neo4j/internal/bolt/path.go

```diff
@@ -62,16 +63,17 @@ func buildPath(nodes []dbtype.Node, relNodes []*relNode, indexes []int) dbtype.P
 		n2 := nodes[n2i]
 
 		rel := dbtype.Relationship{
-			Id:    reln.id,
-			Type:  reln.name,
-			Props: reln.props,
+			Id:        reln.id,
+			ElementId: reln.elementId,
+			Type:      reln.name,
+			Props:     reln.props,
 		}
 		if n1start {
-			rel.StartId = n1.Id
-			rel.EndId = n2.Id
+			rel.StartId, rel.EndId = n1.Id, n2.Id
+			rel.StartElementId, rel.EndElementId = n1.ElementId, n2.ElementId
 		} else {
-			rel.StartId = n2.Id
-			rel.EndId = n1.Id
+			rel.StartId, rel.EndId = n2.Id, n1.Id
+			rel.StartElementId, rel.EndElementId = n2.ElementId, n1.ElementId
```

Comment:
I think it could be a good idea add some tests coverage here.

Reply:
there is in `hydrator_test`